codespell_lib/data/dictionary.txt

```diff
@@ -1283,6 +1285,8 @@ acessories->accessories
 acessors->accessors, accessor,
 acessory->accessory
 acftually->factually, actually,
+achalary->actually
+achaly->actually
```

Comment:
Are these typos you have seen on a regular basis? They look like exceptional typos.

In any case, I would add [`achilary`](https://www.merriam-webster.com/dictionary/achilary) as a fix.